website/docs/r/cdn_frontdoor_rule.html.markdown

```diff
@@ -197,7 +197,9 @@ An `url_redirect_action` block supports the following:
 
 A `route_configuration_override_action` block supports the following:
 
-* `cache_duration` - (Required) When Cache behavior is set to `Override` or `SetIfMissing`, this field specifies the cache duration to use. The maximum duration is 366 days specified in the `d.HH:MM:SS` format(e.g. `365.23:59:59`). If the desired maximum cache duration is less than 1 day then the maximum cache duration should be specified in the `HH:MM:SS` format(e.g. `23:59:59`).
+->**NOTE:** While `cache_duration`, `cache_behavior` and `query_string_caching_behavior` do have default values, if you do not define these properties in your configuration file you will receive a diff during plan.
```

Comment:
this is the default behaviour for Terraform properties, perhaps what we should be saying is something along the lines of: "you can use [Terraform's `ignore_changes` functionality](XXX) to ignore these default values`?

Reply:
Fixed.